gradle.properties

```diff
@@ -2,3 +2,4 @@ version=5.3-SNAPSHOT
 publishUrl=file:../marklogic-data-hub/releases
 systemProp.file.encoding=utf-8
 org.gradle.jvmargs='-Dfile.encoding=UTF-8'
+disBaseUrl=http://nodejs.org/dist/
```

Comment:
Should be "distBaseUrl" - can you do a separate PR to update it? I also recommend verbose property names in general - so "nodeDistributionBaseUrl" would be better.